crates/cursor-core/src/auth/mod.rs

```diff
@@ -131,6 +133,10 @@ async fn polling(
     );
 
     let mut interval = IntervalStream::new(2000);
+
+    // I don't know which value is most suitable, just use 20
+    let mut tried = 20;
```

Comment:
Actually, we didn't set a limit for retrying at the very beginning. And I don't think we should. We will take a look at the implementation of Cursor app before we finally decide how to fix this.

Reply:
In Cursor app, they use `setInterval` to request for "/auth/poll", if the request is failed, they will continue to request until get a right response, then they will stop `setInterval`.

Reply:
I think we should follow that behavior, just remove the limitation of retry times. Users can cancel the polling anytime they want.

Reply:
I don't think so,, trapping into loop is not a good action.In Cursor app, people could close the app to kill the loop.I don't know if we could deactivate the plugin to kill the loop.For me, if request is failed in limitation of retry times, it must not be the network problem, we'd better not send request, otherwise, it just wastes time.But this is my personal opinion, you can follow the loop action based on my PR.